core/templates/pages/exploration-editor-page/modal-templates/post-publish-modal.template.html

```diff
@@ -86,10 +86,10 @@ <h3>Awesome!</h3>
       <div ng-click="selectText($event)"><[explorationLink]></div>
       <span ng-show="explorationLinkCopied">Copied<i class="fa fa-check" aria-hidden="true"></i></span>
     </div>
-    <sharing-links flex="100" layout-type="row"
+    <sharing-links layout-type="row"
                    layout-align-type="center center"
                    share-type="exploration"
```

Comment:
Shouldn't these also be one-way bound with `[]`?

Reply:
Not required, right? As we are passing strings. The syntax with `[]` would look like this: `[layout-type]="'row'"` (Notice the with extra quotes). Angular's `[]` binding expects a variable and since it is a constant string we don't have to use `[]`.